Formula/threesdk.rb

```diff
@@ -0,0 +1,140 @@
+      sha256 "1f05a47122542f97028caf78775a095fbe6a2699b5089de8477eb583167d69aa"
+    end
+
+    resource "macholib" do
+      url "https://files.pythonhosted.org/packages/0d/fe/61e8f6b569c8273a8f2dd73921738239e03a2acbfc55be09f8793261f269/macholib-1.14.tar.gz"
+      sha256 "0c436bc847e7b1d9bda0560351bf76d7caf930fb585a828d13608839ef42c432"
+    end
+
+    resource "PyInstaller" do
+      url "https://files.pythonhosted.org/packages/3c/c9/c3f9bc64eb11eee6a824686deba6129884c8cbdf70e750661773b9865ee0/PyInstaller-3.6.tar.gz"
+      sha256 "3730fa80d088f8bb7084d32480eb87cbb4ddb64123363763cf8f2a1378c1c4b7"
+    end
+
+    resource "packaging" do
+      url "https://files.pythonhosted.org/packages/65/37/83e3f492eb52d771e2820e88105f605335553fe10422cba9d256faeb1702/packaging-20.3.tar.gz"
+      sha256 "3c292b474fda1671ec57d46d739d072bfd495a4f51ad01a055121d81e952b7a3"
+    end
+
+    resource "pyparsing" do
+      url "https://files.pythonhosted.org/packages/c1/47/dfc9c342c9842bbe0036c7f763d2d6686bcf5eb1808ba3e170afdb282210/pyparsing-2.4.7.tar.gz"
+      sha256 "c203ec8783bf771a155b207279b9bccb8dea02d8f0c9e5f8ead507bc3246ecc1"
+    end
+
+    resource "pudb" do
+      url "https://files.pythonhosted.org/packages/10/dc/a4933487ea8322336d3827c1193fff0ddd8b3a43eeda3f446f163dacc407/pudb-2019.2.tar.gz"
+      sha256 "e8f0ea01b134d802872184b05bffc82af29a1eb2f9374a277434b932d68f58dc"
+    end
+
+    resource "urwid" do
+      url "https://files.pythonhosted.org/packages/45/dd/d57924f77b0914f8a61c81222647888fbb583f89168a376ffeb5613b02a6/urwid-2.1.0.tar.gz"
+      sha256 "0896f36060beb6bf3801cb554303fef336a79661401797551ba106d23ab4cd86"
+    end
+
+    resource "pytoml" do
+      url "https://files.pythonhosted.org/packages/f4/ba/98ee2054a2d7b8bebd367d442e089489250b6dc2aee558b000e961467212/pytoml-0.1.21.tar.gz"
+      sha256 "8eecf7c8d0adcff3b375b09fe403407aa9b645c499e5ab8cac670ac4a35f61e7"
+    end
+
+    resource "cffi" do
+      url "https://files.pythonhosted.org/packages/05/54/3324b0c46340c31b909fcec598696aaec7ddc8c18a63f2db352562d3354c/cffi-1.14.0.tar.gz"
+      sha256 "2d384f4a127a15ba701207f7639d94106693b6cd64173d6c8988e2c25f3ac2b6"
+    end
+
+    resource "pycparser" do
+      url "https://files.pythonhosted.org/packages/0f/86/e19659527668d70be91d0369aeaa055b4eb396b0f387a4f92293a20035bd/pycparser-2.20.tar.gz"
+      sha256 "2d475327684562c3a96cc71adf7dc8c4f0565175cf86b6d7a404ff4c771f15f0"
+    end
+
+    resource "PyNaCl" do
+      url "https://files.pythonhosted.org/packages/61/ab/2ac6dea8489fa713e2b4c6c5b549cc962dd4a842b5998d9e80cf8440b7cd/PyNaCl-1.3.0.tar.gz"
+      sha256 "0c6100edd16fefd1557da078c7a31e7b7d7a52ce39fdca2bec29d4f7b6e7600c"
+    end
+
+    def install
+      system 'cd install; version=$(git describe --tags); gsed -i "s/_unreleased_/${version}/" threesdk/__init__.py;'
+      system "cd install; pyinstaller 3sdk.spec"
```

Comment:
```suggestion
      chdir "install" do
        inreplace "threesdk/__init__.py", "_unreleased_" version.to_s
        system "pyinstaller 3sdk.spec"
      end
```

Reply:
`inreplace "threesdk/__init__.py", "_unreleased_", version.to_s`
this fails with me with this error
![image](https://user-images.githubusercontent.com/10658229/81834719-9f861980-9541-11ea-8c74-e99f74d99141.png)

Reply:
There's probably more info around this error.

Reply:
sadly no, this is the only related part
![image](https://user-images.githubusercontent.com/10658229/81925392-bfb8e580-95e0-11ea-9a3b-9f329f814c29.png)

Reply:
Apparently you don't need the inreplace since their released tarbal doesn't have that part of the script.

Reply:
gnused is still working